hr_employee_ppe/models/hr_personal_equipment_request.py

```diff
@@ -12,6 +12,9 @@ class HrPersonalEquipmentRequest(models.Model):
 
     def _compute_contains_ppe(self):
         for rec in self:
+            if not rec.line_ids:
```

Comment:
```suggestion
            rec.contains_ppe = False
            for line in rec.line_ids:
                ...
```
Is it possible to add `rec.contains_ppe = False` first line?
I think it will simple and easy code.

Reply:
@Saran440 I have updated my PR with more simple and easy code.
What do you think?

Reply:
@miguel-S73 Can you try it?
```
def _compute_contains_ppe(self):
    for rec in self:
        rec.contains_ppe = bool(rec.line_ids.filtered("is_ppe"))
```

Reply:
@Saran440  I am reluctant to use `filtered` specially in a `compute` function. `filtered` method has performance issues.

Reply:
Ok, I see.